tests/test_commands.py

```
@@ -1334,3 +1334,36 @@ async def test_update_firmware_bytes_empty(test_charger, caplog):
+    assert "Setting mqtt_vehicle_range_miles to True" in caplog.text
+
+    mock_aioclient.post(
+        TEST_URL_CONFIG,
+        status=200,
+        body='{"msg": "OK"}',
+    )
+    with caplog.at_level(logging.DEBUG):
+        await test_charger_new.set_mqtt_vehicle_range_miles(False)
+    assert "Setting mqtt_vehicle_range_miles to False" in caplog.text
+
+    with pytest.raises(TypeError, match="Value must be a boolean."):
```

Ruff (0.15.18)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)